source/_docs/http-to-https.md

```diff
@@ -60,8 +60,33 @@ Use [Terminus](/docs/terminus) to run `wp search-replace` to converts URLs from
 ```
 terminus remote:wp <site>.<env> -- search-replace 'http://www.example.com' 'https://www.example.com' --all-tables --verbose
 ```
+
 #### Drupal
-Drupal 7 sites can use [Drush Search and Replace (sar)](https://www.drupal.org/project/sar) by [adding custom Drush command](/docs/drush/#add-custom-drush-commands). Drupal 8 sites can use the [Entity API](https://www.drupal.org/docs/8/api/entity-api/introduction-to-entity-api-in-drupal-8) to fetch data from entities in the database that may include insecure references.
+Drupal 7 sites can use [Drush Search and Replace (sar)](https://www.drupal.org/project/sar) by [adding custom Drush command](/docs/drush/#add-custom-drush-commands).
+
+Drupal 8 sites can use the [Entity API](https://www.drupal.org/docs/8/api/entity-api/introduction-to-entity-api-in-drupal-8) to fetch data from entities in the database that may include insecure references.
+
+**Via Dashboard**
```

Comment:
I think this entire section should go above the Drupal header, as an alternative for WP users to search and replace via Pantheon Site Dashboard GUI

Reply:
Confirmed. Drupal sites won't have the ability to clean up DB in the Site Dashboard on Pantheon. That's WP specific 
<img width="759" alt="screen shot 2017-10-20 at 1 57 50 pm" src="https://user-images.githubusercontent.com/10119525/31837506-ad9e469e-b59e-11e7-9b83-0407c1abdaff.png">